homeassistant/components/zerproc/light.py

```diff
@@ -94,6 +78,11 @@ async def discover(*args):
     async_track_time_interval(hass, discover, DISCOVERY_INTERVAL)
 
 
+async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Cleanup the Zerproc integration."""
+    hass.data.pop(DOMAIN, None)
```

Comment:
This clean up needs to move to the `async_unload_entry` function in `__init__.py` module. The platforms don't have this interface.

Reply:
Wow, not sure how I missed that. I assumed `async_forward_entry_unload` mirrored the behavior of `async_forward_entry_setup` and called the methods on the platform. Thanks! I pushed up a fix and added tests this time.